CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 [...]
 - **[NEW]** Add `Marquee`
+- **[NEW]** Add `TripCardSample`
```

Comment:
I may suggest **TripPrice**

Reply:
I am still not sure what's right, I'll keep `TripCardSample` for now